### DISCUSSION TITLE
Dynamic Import rerender since 9.1.2

### DISCUSSION THREAD

**Discussion by Mohsan1995** [vercel/next.js#12341] [Help]
Since the migration of next 9.1.2, I have many issue with the dynamic import : 
1) My componenets re-render in the cllent side. I had the element generated in server side, the show up in client side, than dissapear in the client and than reapper.
2) Issue with the layout in dynamic. I have a footer in css in js and the css is not apply than the component are in dynamic.

Do you have any clue about the change of the dynamic, and his new behavior ?

**Comment by kokipsiho** [comment]
See this https://github.com/zeit/next.js/discussions/11393, I have similar issues like you. It was introduced with the change I wrote about. 
Next dynamic import is unusable as it is now, something must be done about it.

**Reply by Mohsan1995** [reply]
Yes, I read I had similar issue with the concurrent mode. I will try to create a sample in a different repo for create the same behavior. 
So we have no clue about the issue ?

**Comment by drobledom** [comment]
Did you find a solution for this? Same is happening to us in v12
